tests/core_tests/test_api.py

```diff
@@ -506,6 +507,125 @@ def test_project_path_property(self):
         self.assertEqual(self.tk.project_path, self.project_root)
 
 
+class TestDefaultStorageRootHook(TankTestBase):
```

Comment:
Could we move this class to a dedicated class regarding testing the new hook?
Maintaining tests is easier with small files.